frontend/src/component/menu/Header/NavigationLink/NavigationLink.tsx

```diff
@@ -1,9 +1,13 @@
 import { ListItem, Link, styled } from '@mui/material';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { EnterpriseBadge } from 'component/common/EnterpriseBadge/EnterpriseBadge';
+import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
 import { Link as RouterLink } from 'react-router-dom';
 interface INavigationLinkProps {
     path: string;
     text: string;
     handleClose: () => void;
+    mode?: ('pro' | 'enterprise')[];
```

Comment:
is it the only place we use this type? seems quite generic. Why is mode getting an array? Is it modes - plural?